estatioapp/app/src/main/java/org/estatio/module/party/dom/relationship/Party_PartyRelationshipContributions.java

```diff
@@ -24,20 +24,22 @@
 import javax.annotation.PostConstruct;
```

Comment:
The changes made are subsequently reverted.  I suspect this was because of the issues.

Suggest you un-revert and fix forward by applying those review comments.

Reply:
This was reverted because the PartyRelationshipViewService_IntegTest was failing for parentChild() and childParent() (it's trying to call the relationships(Party party) method. How do I fix this integ test to use the mixin correctly?

Reply:
to call a mixin, use factoryService to instantiate the mixin.  It may also need to be wrapped.

To create a mixin and invoke without a wrapper, It's something like:

factoryService.mixin(target, MixinClass.class).act(...)

To invoke with a wrapper, it's

wrapperFactory.wrap( factoryService.mixin(...) ).act(...)

I think that wrap() and mixin() are both in the superclass as convenience methods, so this becomes 

wrap(mixin(target, MixinClass.class)).act(...)